src/rgw/rgw_kafka.cc

```diff
@@ -412,12 +426,10 @@ class Manager {
     if (rc == -1) {
       const auto err = rd_kafka_last_error();
       ldout(conn->cct, 10) << "Kafka publish: failed to produce: " << rd_kafka_err2str(err) << dendl;
-      // TODO: dont error on full queue, and don't destroy connection, retry instead
       // immediatly invoke callback on error if needed
       if (message->cb) {
-        message->cb(err);
+        message->cb(-rd_kafka_err2errno(err));
```

Comment:
can we also make the log_level =1 for line no. 428, since its an error and would like to know the kafka error